docs/_Research-Data-Management/03-md.md

```diff
@@ -4,14 +4,49 @@ category: Research-Data-Management
 layout: default
 docs_css: markdown
 redirect_from: /Research-Data-Management
-empty: true
-hide: true
+empty: false
+hide: false
 ---
 
 # Metadata
 
+## Metadata is data about data
+
+Before we delve into specifications on what Metadata standards for the Microbiology community are or would be, let us explain what metadata is. 
+
+Metadata is information that describes other information in order to help you understand or use it. In the simplest terms, metadata is **data** about data. So, in general, metadata provides us with information about other data but does not tell us anything about the content of the data itself.
+
+If you find our explanation lacking, we refer you to the [FAIR Cook Book](https://faircookbook.elixir-europe.org/content/home.html), more specifically to the [FAIR and the notion of metadata](https://faircookbook.elixir-europe.org/content/recipes/introduction/metadata-fair.html) section.
+
+## When should you collect your metadata
+
+As is usually the case in sciences, your research (and the wider microbiological community) can benefit highly from the rigorous and timely planning of your experiments, including metadata collection. In this case, we refer you to other subsections of this KnowledgeBase: [**Data Management Plans (DMPs)**](./08-dmp.md) that could help you plan your experiments.
+
+Metadata collection should be planned, but at the same time, it can be overwhelming. What amount of metadata is enough to describe your data? Was it crucial to note down the pipette I was using, or should I have noted down the location of the sampling site? Will one year older me understand what I wrote down in my notebook/ELN? Will other researchers make sense of the (meta)data I collected? Will other researchers be able to replicate my research if I did not note down my in-house DNA extraction protocol? 
```

Comment:
```suggestion
Metadata collection should be planned, but at the same time, it can be overwhelming. What amount of metadata is enough to describe your data? Was it crucial to note down the pipette I was using, or should I have noted down the location of the sampling site? Will I still understand in a year what I wrote down in my notebook/ELN? Will other researchers make sense of the (meta)data I collected? Will other researchers be able to replicate my research if I did not note down my in-house DNA extraction protocol? 
```